.estafette.yaml

```diff
@@ -19,6 +19,10 @@ stages:
       GOOS: linux
       GOGC: off
     commands:
+    - apk -U add curl
+    - curl -sL https://go.kubebuilder.io/dl/2.2.0/linux/amd64 | tar -xz -C /tmp/
+    - mv /tmp/kubebuilder_2.2.0_linux_amd64 /usr/local/kubebuilder
+    - export PATH=$PATH:/usr/local/kubebuilder/bin
```

Comment:
Yup it's nasty, but first let's get things to work before creating any helper images.